### ISSUE TITLE
postcss 8 breaks this? 

### ISSUE THREAD

**Issue by jasikpark** [leodido/postcss-clean#40]
```
TypeError: node.getIterator is not a function
    at LazyResult.visitTick (/Users/calebjasik/Desktop/Github/live.collegemusic.co.uk/node_modules/postcss/lib/lazy-result.js:484:33)
    at LazyResult.runAsync (/Users/calebjasik/Desktop/Github/live.collegemusic.co.uk/node_modules/postcss/lib/lazy-result.js:372:30)
    at processTicksAndRejections (node:internal/process/task_queues:93:5)
    at async Promise.all (index 0)
error Command failed with exit code 1.
```

https://github.com/jasikpark/live.collegemusic.co.uk/pull/12/commits/b6168626b91237264778506bd51f63d171fded96

**Comment by RiZKiT** [comment]
I'm afraid this repo is abandoned and the pull request will not be merged. 😕

**Comment by leodido** [comment]
#41 got merged!

**Comment by simonsarris** [comment]
This issue seems back, at least with:

    "postcss": "^8.2.4",
    "postcss-clean": "^1.2.1",
    "postcss-cli": "^8.3.1",

Has anyone else experienced a regression?

**Comment by leodido** [comment]
Yes, I confirm and it was wanted.

I published 1.2.1 to keep backward compatibility. Will republish support for postcss 8 on a new major release (eg., 2.0.0).
